### ISSUE TITLE
Cross-portal block placement direction abnormal

### ISSUE THREAD

**Issue by mapleSyrup982** [qouteall/ImmersivePortalsMod#875]
TL;DR: Pistons face wrong with portals turned 90 degrees

I was working on a redstone machine with the portals on, and while placing pistons I was confused what was happening when placing the pistons as they won't point where I wanted.

I realized that this problem was because I was placing it through a portal to where I was facing towards negative X (west), and since I was facing west, the piston would point east. the portal was turned 90 degrees so this looked off. I had wanted it to point north when seen from said location on the other side of the portal.
(I'm sorry if this doesn't make sense, English isn't my best strength)

This is also the case with other blocks, such as levers, redstone repeaters, banners, and stairs.

Here is my full mods list:

- fabric-api-0.40.1+1.17.jar
- EntityCulling-Fabric-1.3.3.jar
- iris-and-sodium-mc1.17-1.1.2+build.9.jar_
- lithium-fabric-mc1.17.1-0.7.4.jar

Here is the glitch:
![ImmersivePortalsGlitchPicture2](https://user-images.githubusercontent.com/86534266/140629889-a1cfffc5-78fd-415b-a3af-ad89567e147e.png)

F3 menu:
![ImmersivePortalsPicture3](https://user-images.githubusercontent.com/86534266/140629900-1b0b8940-a315-4774-be48-a310aaf376be.png)

What it should look like w/ F3:
![ImmersivePortalsPicture4](https://user-images.githubusercontent.com/86534266/140629915-dcbe23b3-1a38-4166-8700-fb69b0aa8a6f.png)
